src/encord_active/cli/main.py

```diff
@@ -61,6 +63,7 @@ def list_commands(self, ctx: click.Context):
 cli.add_typer(config_cli, name="config", help="[green bold]Configure[/green bold] global settings 🔧")
 cli.add_typer(import_cli, name="import", help="[green bold]Import[/green bold] Projects or Predictions ⬇️")
 cli.add_typer(print_cli, name="print")
+cli.add_typer(metric_cli, name="metric", help="[green bold]Manage[/green bold] project's metrics.")
```

Comment:
I think it would make sense to move the `metricize` command into `metrics` and call it `run`

Reply:
perfect, I'm fond of doing that also 🍀.